maplibre-winit/src/input/pan_handler.rs

```diff
@@ -25,7 +25,9 @@ impl UpdateState for PanHandler {
                 (self.window_position, self.start_window_position)
             {
                 let view_proj = view_state.view_projection();
-                let inverted_view_proj = view_proj.invert();
+                let Some(inverted_view_proj) = view_proj.invert() else {
+                    todo!("what to do here?")
```

Comment:
Basically in this case we would skip the execution of `update_state`. Apparently it can happen for zero-sized windows. So this should be handled here either with ignoring it or doing a `.except()`. You usually can not click in a zero-sized window. The except should explain that.